modules/broadcast/src/backend/db.ts

```diff
@@ -191,7 +192,7 @@ export default class BroadcastDb {
       .where(function() {
         this.where(isPast).andWhere('broadcast_outbox.botId', botId)
       })
-      .join('srv_channel_users', 'srv_channel_users.user_id', 'broadcast_outbox.userId')
+      .join('srv_channel_users', 'srv_channel_users.id', 'broadcast_outbox.userId')
```

Comment:
I think it would be better to keep user_id, since it's the value we use everywhere to reference a user. The numeric ID has no value outside that table